action/ls.go

```diff
@@ -107,15 +106,15 @@ func (lh *ListHandler) WithZone(z string) *ListHandler {
 
 // ListObjects will handle all ls actions.
 func (lh *ListHandler) ListObjects() (err error) {
-	bucketName, objectKey, err := task.ParseQsPath(lh.Remote)
+	_, objectKey, err := utils2.ParseQsPath(lh.Remote)
 	if err != nil {
 		return err
 	}
 
-	err = contexts.Storage.SetupBucket(bucketName, lh.Zone)
-	if err != nil {
-		return
-	}
+	// err = stor.SetupBucket(bucketName, lh.Zone)
```

Comment:
Why cancel setup bucket here?

Reply:
Don't need to review action package, it will be removed.